configure

```diff
@@ -690,7 +690,8 @@ def configure_library(lib, output):
     if default_libpath:
       default_libpath = '-L' + default_libpath
     (pkg_libs, pkg_cflags, pkg_libpath) = pkg_config(lib)
-    cflags = pkg_cflags.split('-I') if pkg_cflags else default_cflags
+    # Remove empty strings from the list of include_dirs
+    cflags = filter(None, pkg_cflags.split('-I')) if pkg_cflags else default_cflags
```

Comment:
I haven't come across that just yet. I'll improve the filter; guess it couldn't hurt. Just want to avoid making parsing too complex.